### PR TITLE
Remove template literals from "escaping characters" section

### PR DESCRIPTION
**Pull request by collin-klippel**
### Description

Removes unrelated mention of template literals from section on escaping characters.

### Motivation

Avoid confusing readers.

### CODE REVIEW

File: files/en-us/web/javascript/guide/grammar_and_types/index.md [hunk 1]
```diff
@@ -778,16 +778,6 @@ lines.'
 console.log(str);   // this string is broken across multiple lines.
 ```
 
-There is also a [**template literal**](/en-US/docs/Web/JavaScript/Reference/Template_literals) syntax. This allows for many advanced text formatting use cases, including multiline strings!
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/guide/grammar_and_types/index.md:781]: I agree this looks weird, but I think the intention is that if you use template literals, you don't need to escape line breaks at all—maybe just reduce this paragraph to a sentence?

Comment (collin-klippel) [reply files/en-us/web/javascript/guide/grammar_and_types/index.md:781]: Yeah I agree! I only realized the intention after the fact lol. Let's see...

File: files/en-us/web/javascript/guide/grammar_and_types/index.md [hunk 2]
```diff
@@ -778,7 +778,7 @@ lines.'
...
+But if this is your goal, there is also [**template literal**](/en-US/docs/Web/JavaScript/Reference/Template_literals) multiline string syntax where you don't need to escape line breaks.
```
Comment (collin-klippel) [inline files/en-us/web/javascript/guide/grammar_and_types/index.md:781]: I guess my thinking here is that I think it'd be nice if this copy specifically referenced the previous code sample 🤔 I think it helps tamper the non-sequitur vibe. Huge opinion though, I'm not really sold on this one myself 😅

File: files/en-us/web/javascript/guide/grammar_and_types/index.md [hunk 3]
```diff
@@ -778,7 +778,7 @@ lines.'
 console.log(str);   // this string is broken across multiple lines.
 ```
 
-There is also a [**template literal**](/en-US/docs/Web/JavaScript/Reference/Template_literals) syntax. This allows for many advanced text formatting use cases, including multiline strings!
+But if this is your goal, there is also [**template literal**](/en-US/docs/Web/JavaScript/Reference/Template_literals) multiline string syntax where you don't _need_ to escape line breaks.
```
Comment (collin-klippel) [inline files/en-us/web/javascript/guide/grammar_and_types/index.md:781]: I guess my thinking here is that I think it'd be nice if this copy specifically referenced the previous code sample 🤔 I think it helps tamper the non-sequitur vibe. Huge opinion though, I'm not really sold on this one myself 😅

Comment (Josh-Cena) [inline files/en-us/web/javascript/guide/grammar_and_types/index.md:781]: I don't think we need the code example here at all—you are right, it feels really non-sequitur. I suggest deleting the code block following this and changing this to:

```suggestion
If you use [template literals](/en-US/docs/Web/JavaScript/Reference/Template_literals) introduced above to write multiline strings, you don't need to escape line breaks at all.
```

Comment (collin-klippel) [reply files/en-us/web/javascript/guide/grammar_and_types/index.md]: 🫡

File: files/en-us/web/javascript/guide/grammar_and_types/index.md (hunk 1 above)
Comment (collin-klippel) [reply files/en-us/web/javascript/guide/grammar_and_types/index.md:781]: Okay so I was starting at this for a bit and I realized that these two code blocks actually represent entirely different use cases. The first is breaking over multiple lines a string that is supposed to print on one line, which is different from multiline strings.

I think it's pretty confusing–perhaps we should remove entirely?

Comment (Josh-Cena) [inline files/en-us/web/javascript/guide/grammar_and_types/index.md:781]: Ummm, I think we are keeping this paragraph (with the reword suggested above)?

Comment (collin-klippel) [reply files/en-us/web/javascript/guide/grammar_and_types/index.md:781]: Ah sorry I commented on a resolved chain 🤦 

I was starting at this for a bit and I realized that these two code blocks actually represent entirely different use cases. The first is breaking over multiple lines a string that is supposed to print on one line, which is different from multiline strings where a string written over multiple lines actually prints over multiple lines.

I think it's pretty confusing–perhaps we should remove entirely?

Comment (Josh-Cena) [reply files/en-us/web/javascript/guide/grammar_and_types/index.md:781]: Ah, you are right—didn't think about that! Yes, the template literal equivalent would be [`deline`](https://www.npmjs.com/package/deline).

**Review by Josh-Cena — APPROVED** [review]
Thank you!

**Pull request merged**